aws_lambda_builders/workflows/nodejs_npm_esbuild/workflow.py

```diff
@@ -78,13 +80,15 @@ def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtim
             )
 
         # if we're building in the source directory, we don't have to copy the source code
-        self.actions = (
+        self.actions: List[BaseAction] = (
             []
             if self.build_dir == self.source_dir
             else [CopySourceAction(source_dir=self.source_dir, dest_dir=self.build_dir, excludes=self.EXCLUDED_FILES)]
         )
 
         if self.download_dependencies:
+            # Manifest is different from the one in the project directory
+            specify_manifest = str(Path(manifest_path).parent) != source_dir
```

Comment:
Is `manifest_path` specified all the time, even without `--manifest` option?

Reply:
Yes